corehq/apps/domain/forms.py

```diff
@@ -789,9 +789,20 @@ class PrivacySecurityForm(forms.Form):
     restrict_superusers = BooleanField(
         label=ugettext_lazy("Restrict Dimagi Staff Access"),
         required=False,
-        help_text=ugettext_lazy("Dimagi staff sometimes require access to projects to provide support. Checking "
-                                "this box may restrict your ability to receive this support in the event you "
-                                "report an issue. You may also miss out on important communications and updates.")
+        help_text=ugettext_lazy(
+            "CommCare support staff sometimes require access "
```

Comment:
```suggestion
            "CommCare support staff sometimes requires access "
```

I think staff is considered singular right? Though I'm second guessing myself

Reply:
I think this is analogous to "A group of us are (pl) going" vs. "The group has (sg) decided" where either could b appropriate depending on the context. (https://abcbusinesswriting.blogspot.com/2009/08/staff-is-vs-staff-are.html). Honestly I think in this case either would be fine, but since I'm referring to the individuals that make up the staff, I'll stick with plural here